Lib/platform.py

```diff
@@ -238,12 +238,7 @@ def _norm_version(version, build=''):
     l = version.split('.')
     if build:
         l.append(build)
-    try:
-        ints = map(int, l)
-    except ValueError:
-        strings = l
-    else:
-        strings = list(map(str, ints))
+    strings = list(map(str, map(int, l)))
```

Comment:
We'll still need the `ValueError` check around this.

```suggestion
    try:
        strings = list(map(str, map(int, l)))
    except ValueError:
        strings = l
```

Reply:
Forgot about that. Thanks for reminding.